gamestonk_terminal/cryptocurrency/defi/graph_model.py

```diff
@@ -0,0 +1,271 @@
+"""The Graph model"""
+__docformat__ = "numpy"
+
+import datetime
+import requests
+import pandas as pd
+
+from cryptocurrency.dataframe_helpers import very_long_number_formatter
+
+UNI_URL = "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2"
+
+
+def query_graph(url: str, query: str) -> dict:
+    """Helper methods for querying graphql api. [Source: https://thegraph.com/en/]
+
+    Parameters
+    ----------
+    url: str
+        Endpoint url
+    query: str
+        Graphql query
+
+    Returns
+    -------
+    dict:
+        Dictionary with response data
+    """
+
+    request = requests.post(url, json={"query": query})
+    if request.status_code == 200:
+        return request.json()["data"]
+    return {}
+
+
+def get_uni_tokens(skip: int = 0, limit: int = 100) -> pd.DataFrame:
+    """Get list of tokens trade-able on Uniswap DEX. [Source: https://thegraph.com/en/]
+
+    Parameters
+    ----------
+    skip:
+        Skip n number of records.
+    limit: int
+        Show n number of records.
+
+    Returns
+    -------
+    pd.DataFrame
+        Uniswap tokens with trading volume, transaction count, liquidity.
+    """
+
+    query = """
+            {
+            tokens(first: %s, skip:%s) {
+                symbol
+                name
+                tradeVolumeUSD
+                totalLiquidity
+                txCount
+                }
+            }
+        """ % (
```

Comment:
I think we are normally using f formatted strings, should we refactor this to use it due to consistency? @DidierRLopes

Reply:
Yup, f string is much better for readability